src/utils/moviesSvc.js

```diff
@@ -0,0 +1,34 @@
+const base = `http://localhost:4000`;
+
+const call = (path, options = {}) => {
+  const mergedOptions = {
+    headers: { 'Content-Type': 'application/json' },
+    ...options
+  };
+
+  return new Promise(async (resolve) => {
+    await fetch(`${base}${path}`, mergedOptions)
+      .then(res => res.json())
+      .then(json => resolve(json))
+      .catch(e => {
+        console.log(e);
+        resolve();
+      });
+  });
+}
```

Comment:
are you sure you need Promise and async as well?

Reply:
oh, right, fetch returns promise by itself